NAMESPACE

```diff
@@ -42,6 +42,7 @@ export(write_nml)
 import(GLMr)
```

Comment:
I think keeping this `import(GLMr)` swamps the function-specific import you added below. Or at least that is my understanding of the namespacing